services/management/external.go

```diff
@@ -111,6 +110,12 @@ func (e *ExternalService) AddExternal(ctx context.Context, req *managementpb.Add
 			return err
 		}
 
+		if !req.SkipConnectionCheck {
+			if err = e.registry.CheckConnectionToService(ctx, tx.Querier, service, row); err != nil {
```

Comment:
We shouldn't send connection check request to pmm clients with version lower then 2.15.0, it will raise panic there.
https://github.com/percona/pmm-agent/pull/237/files#diff-6eb9fa169fdb33f5cf7bf4a8267eef04668d3cee3ccb099593589a30067c141aR82

Reply:
@askomorokhov Can you please fix it in scope of CheckConectionToService? To cover also haproxy.